ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node.hs

```diff
@@ -220,12 +216,16 @@ data LowLevelRunNodeArgs m addrNTN addrNTC versionDataNTN versionDataNTC blk
                         -> m a
 
       -- | The " static " ChainDB arguments
-    , llrnChainDbArgsDefaults :: ChainDbArgs Defaults m blk
+    , llrnChainDbArgsDefaults :: Incomplete ChainDbArgs m blk
+
+      -- | FS on which the directories for the different databases will be
+      -- created.
+    , llrnMkHasFS :: ChainDB.RelativeMountPoint -> SomeHasFS m
```

Comment:
What is it exposed for? Would we want to use `stdMkChainDbHasFS` by default instead of exposing this function?

If there is a specific reason that we're exposing this new function, then maybe we should either mention that here or in the changelog fragment

Reply:
Well, it is defined because it will need to be used by UTXO-HD to create the normal FS and the ssd FS, but the node uses the StdArgs and not the LLRNArgs